spring-cloud-gcp-starters/spring-cloud-gcp-starter-core/src/main/java/org/springframework/cloud/gcp/core/autoconfig/CredentialsAutoConfiguration.java

```diff
@@ -32,12 +32,11 @@ public void setEnvironment(Environment environment) {
 	public GoogleCredentials googleCredentials(ResourceLoader resourceLoader)
 			throws Exception {
 		if (!StringUtils.isEmpty(this.environment.getProperty(AUTH_LOCATION_KEY))) {
-			return GoogleCredentials.fromStream(new FileInputStream(resourceLoader
-					.getResource(this.environment.getProperty(AUTH_LOCATION_KEY))
-					.getFile()));
-		}
-		else {
-			return GoogleCredentials.getApplicationDefault();
+			return GoogleCredentials.fromStream(
+					new FileInputStream(resourceLoader.getResource(
+							this.environment.getProperty(AUTH_LOCATION_KEY)).getFile()));
```

Comment:
OK, https://github.com/spring-cloud/spring-cloud-gcp/issues/31. Just wanted to pay attention at the moment when I have noticed suspicious code